server/computer_use/utils.py

```diff
@@ -96,11 +95,11 @@ def _inject_prompt_caching(
         ):
             if breakpoints_remaining:
                 breakpoints_remaining -= 1
-                content[-1]['cache_control'] = BetaCacheControlEphemeralParam(
```

Comment:
seems to be anthropic specific, best move it there or at least make it clear by name